tap_twilio/transform.py

```diff
@@ -22,14 +22,20 @@ def subresources_to_array(data_dict, data_key):
 
 
 def deserialise_jsons_in_dict(data_dict, jsons_keys):
-    for jsons_key in jsons_keys:
-        if jsons_key in data_dict:
-            # Deserialise only string types
-            if isinstance(data_dict[jsons_key], str):
-                try:
-                    data_dict[jsons_key] = json.loads(data_dict[jsons_key])
-                except json.JSONDecodeError:
-                    data_dict[jsons_key] = {'invalid_json': data_dict[jsons_key]}
+    if isinstance(data_dict, dict):
+        # Recursive deserialize JSON string child items
+        for key in data_dict.keys():
+            data_dict[key] = deserialise_jsons_in_dict(data_dict[key], jsons_keys)
+
+        if isinstance(data_dict, dict):
```

Comment:
we don't need this check, since in line 25 we already made sure `data_dict` is a dict, right?

Reply:
thanks, fixed by https://github.com/transferwise/pipelinewise-tap-twilio/pull/23/commits/145211f2f066e9d50f26dbb86168353826569fab